### PR TITLE
Add auth logic and squash bugs

### PR DESCRIPTION
**Pull request by allinthemind66**
This PR adds some auth logic around routes, as well as fixes some errors/bugs/warnings.

### CODE REVIEW

File: src/component/signUpPage/signUpPage.js [hunk 1]
```diff
@@ -1,41 +1,70 @@
...
+                    <input type="text" id="lastName" placeholder="Last name" value={lastName} onChange={e => setLastName(e.target.value)} /><br />
+                    <label >Email</label><br />
+                    <input type="text" id="email" placeholder="Email" value={email} onChange={e => setEmail(e.target.value)} /><br />
+                    <label>Password</label><br />
+                    <input type="password" placeholder="Password" value={password} onChange={e => setPassword(e.target.value)} /><br />
+                    <label>Confirm Password</label><br />
+                    <input type="password" placeholder="Confirm password" value={confirmPassword} onChange={e => setConfirmPassword(e.target.value)} /><br /><br />
+                    <button className="button" onClick={handleOnSubmit}>Submit</button>
+                </form>
+                <div className="signInText">Already a user? Sign in <Link to="/">here!</Link></div>
```
Comment (allinthemind66) [inline src/component/signUpPage/signUpPage.js:64]: Placeholder link for now, until we get the signin page pushed

File: src/App.js [hunk 2]
```diff
@@ -8,35 +8,59 @@ import Meeting from './component/Meeting/meeting';
 
 //Rick
 import SignUpPage from './component/signUpPage/signUpPage';
+import ProtectedRoute from './component/protectedRoute'
 
 //Yi-Chun
-import SearchPage from './component/SearchPage/SearchPage';
+// import SearchPage from './component/SearchPage/SearchPage';
```
Comment (lukegehron) [inline src/App.js:14]: lol, wut? no more search page?

File: src/App.js [hunk 3]
```diff
@@ -8,35 +8,59 @@ import Meeting from './component/Meeting/meeting';
...
-        <Route path="/Meeting">
-          <Meeting />
-        </Route>
-        <Route path="/">
-          <Meeting />
-        </Route>
-        <Route path="/manageTutorTimes">
-          <TutorSubjectsAndTimes />
-        </Route>
+        <ProtectedRoute
```
Comment (lukegehron) [inline src/App.js:40]: whats the differencee between Route and ProtectedRoute?

File: src/component/Meeting/meeting.js [hunk 4]
```diff
@@ -74,12 +74,12 @@ const Meeting = () => {
         </Col>
         <Col md={{ span: 10, offset: 0 }}>
           <div className='p5'>
-            <div class="chat-popup" id="myForm">
```
Comment (lukegehron) [inline src/component/Meeting/meeting.js:77]: nice catch, thanks

File: src/component/protectedRoute.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
+/**
+ * This component is a reusable, authenticated route. By using this, we will block any other routes until the user is signed in
+ */
```
Comment (lukegehron) [inline src/component/protectedRoute.js:3]: ah, protectedRoute is ours, not react-route. got it. nice

**Review by lukegehron — APPROVED** [review]
all looks good to me, except commenting out Yi-Chuns section

File: src/App.js (hunk 2 above)
Comment (allinthemind66) [reply src/App.js:14]: It wasn't being used anywhere, so I commented it out because it was causing errors in the console

Comment (allinthemind66) [reply src/App.js:14]: We can comment back in once it's being used

Comment (lukegehron) [reply src/App.js:14]: I think we just need to add 
<Route path="/SearchPage">
          <SearchPage />
        </Route>

**Pull request closed without merging**